src/ispc/kernels/lanczos3.ispc

```diff
@@ -18,97 +18,190 @@ static inline float sinc(float x)
     return sin(x) / x;
 }
 
-static inline float lanczos3_filter(float t)
+static inline float lanczos3_filter(float t, uniform float filter_scale)
 {
     t = abs(t);
 
-    if (t < 3.0)
-        return clean(sinc(t) * sinc(t / 3.0));
+    if (t < filter_scale)
+        return clean(sinc(t) * sinc(t / filter_scale));
     else
         return 0.0f;
 }
 
-struct Image {
-    uniform const uint8* data;
-    uniform int<2> size;
+struct WeightVariables {
```

Comment:
Do we have a more descriptive name than "variables"? This has to do with image size/resolution, right?